packages/Select/__tests__/Select.spec.jsx

```diff
@@ -6,9 +6,9 @@ import Text from '@tds/core-text'
 import Paragraph from '@tds/core-paragraph'
 import InputFeedback from '@tds/core-input-feedback'
 import DecorativeIcon from '@tds/core-decorative-icon'
+import { Fade } from '@tds/shared-animation'
```

Comment:
Was this reflected in its package.json?

Reply:
Yes: https://github.com/telus/tds-core/blob/2ae6bdb60a46f22854c5ee965379bf0597bdb640/packages/Select/package.json#L38

Reply:
From this commit hash, I see that it is still in `devDependencies`

https://github.com/telus/tds-core/blob/2ae6bdb60a46f22854c5ee965379bf0597bdb640/packages/Select/package.json#L36-L38

Reply:
My mistake, this should remain in `devDependencies` since it's only being used in tests.